build.gradle

```diff
@@ -91,7 +91,7 @@ allprojects {
         trimTrailingWhitespace()
 
         endWithNewline()
-        googleJavaFormat()
+        googleJavaFormat('1.7')
```

Comment:
pin code style version to what it was before upgrade, otherwise spotless suggests minor indentation changes